src/sentry/tasks/sentry_functions.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+from sentry.tasks.base import instrumented_task
+from sentry.utils import json
+from sentry.utils.cloudfunctions import publish_message
+
+TASK_OPTIONS = {
+    "queue": "app_platform",
```

Comment:
@brianthi @AniketDas-Tekky Is it OK for Sentry functions to use the same queue as the rest of the integrations?

Reply:
Should be fine. Let's keep an eye on queue times